packages/teams-js/src/internal/utils.ts

```diff
@@ -54,8 +70,9 @@ export function validateOrigin(messageOrigin: URL): boolean {
     return false;
   }
   const messageOriginHost = messageOrigin.host;
+  const validDomains = GlobalVars.validOrigins.length !== 0 ? GlobalVars.validOrigins : validOriginsFallback;
```

Comment:
>const validDomains = GlobalVars.validOrigins.length !== 0 ? GlobalVars.validOrigins : validOriginsFallback; [](http://example.com/codeflow?start=2&length=107)

Not sure if it belongs here or elsewhere (given some other feedback about the design) but it is important to write log statements to make it clear to any troubleshooters exactly where the valid origins its using came from (CDN URL or fallback)

Reply:
Adding in a separate parameter to validateOriginLogger that includes which list it is being compared to.